.gitignore

```diff
@@ -3,6 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
It is a good practice to ignore database files like `db.sqlite3` to prevent committing sensitive data or large files to the repository. Ensure that this line is correctly placed in the `.gitignore` file.